keyboards/sofle/keymaps/noqmk/keymap.c

```diff
@@ -0,0 +1,686 @@
+#include QMK_KEYBOARD_H
```

Comment:
This needs a license header (https://docs.qmk.fm/#/pr_checklist?id=example-gplv2-header) at the top.